packages/scandipwa/src/component/ProductAttributeValue/ProductAttributeValue.component.js

```diff
@@ -269,20 +269,34 @@ export class ProductAttributeValue extends PureComponent {
         );
     }
 
-    getCheckboxLabel = (label, subLabel) => (
-        <span
-          block="ProductAttributeValue"
-          elem="Label"
-        >
-            { label }
+    renderSublabel = (subLabel) => {
```

Comment:
Please avoid arrow functions in classes, use bind(this) in this case. There is a problem with making plugins for arrow functions in classes.